source/apickli/apickli-gherkin.js

```diff
@@ -25,38 +26,38 @@ const callbackWithAssertion = function(callback, assertion) {
     }
 };
 
-module.exports = function() {
-    this.registerHandler('BeforeScenario', function(scenario, callback) {
-        stepContext.scenario = scenario.getName;
-        callback();
-    });
+defineSupportCode(function({Before}) {
+    Before(function(scenarioResult, callback) {
+        // https://github.com/cucumber/cucumber-js/issues/891
```

Comment:
let's keep it so people can understand why they are commented. Once the issue is fixed, we can remove it.